team/caesar-tuguinay.yaml

```diff
@@ -1,19 +1,15 @@
 - title: Caesar Tuguinay
   avatar: https://uw-echospace.github.io/author/caesar-tuguinay/avatar_hu91f4814411e1234976d159b9f96b7860_33992_270x270_fill_q90_lanczos_center.jpg
-  role: Your title
+  role: Research Assistant
   organizations:
-    - name: Your institution
-  bio: A short bio about yourself.
-    You can have multiple lines. Just watch that you have two leading spaces at the beginning.
+    - name: Applied Physics Lab, University of Washington
+  bio: Caesar earned a B.S. in Mathematics from the University of Washington in 2023 and is currently pursuing an M.Sc. in Applied Mathematics also at the University of Washington, expecting to graduate in 2025. He is a developer and maintainer of the [Echopype](https://github.com/OSOceanAcoustics/echopype) open-source Python package, which enables scalable echosounder data processing. Using Echopype as a foundation, Caesar conducts research in machine learning applications for fisheries acoustics.
```

Comment:
@ctuguinay the links cannot render in the bio description. At this point I believe it just accepts strings. So can you just remove the link to echopype?

Reply:
Just removed :+1: @valentina-s